frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/ReviewInformation/EditCamperCard.tsx

```diff
@@ -110,9 +135,51 @@ const EditCamperCard = ({
       data: initialCamper.specialNeeds,
     });
 
+    personalInfoQuestions.forEach((question) => {
+      dispatchPersonalInfoAction({
+        type: PersonalInfoActions.UPDATE_RESPONSE,
+        camperIndex,
+        question: question.question,
+        data: initialCamperFormResponses[question.question] || "",
+      });
+    });
+    setSave(false);
     setEditing(false);
   };
 
+  const handleMultipleChoiceChange = (
+    choice: string,
+    question: FormQuestion,
+  ) => {
```

Comment:
I feel like we could make the prop naming more consistent accross these dispatchs - e.g. multipleChoiseResponse, selectionsResponse, textResponse

Reply:
There were actually quite a few inconsistencies regarding this name before, since we use it in quite a few components (notably edit components). I've made the names the same through the different components, so I rather keep it as is right now.